Library/Homebrew/cmd/update.sh

```diff
@@ -471,7 +471,7 @@ EOS
      [[ -d "${HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-core" ]]
   then
     safe_cd "${HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-core"
-    echo "HOMEBREW_CORE_GIT_REMOTE set: using ${HOMEBREW_CORE_GIT_REMOTE} for Homebrew/brew Git remote."
+    echo "HOMEBREW_CORE_GIT_REMOTE set: using ${HOMEBREW_CORE_GIT_REMOTE} for Homebrew/core Git remote."
```

Comment:
This is unrelated to the other changes in this PR but I noticed this and figured it was a typo